src/main/java/appeng/client/ClientHelper.java

```diff
@@ -79,7 +81,11 @@
 	public void preinit()
 	{
 		MinecraftForge.EVENT_BUS.register( this );
-		ModelLoaderRegistry.registerLoader( UVLModelLoader.INSTANCE );
+		// Do not register the Fullbright hacks if Optifine is present or if the Forge lighting is disabled
+		if( !FMLClientHandler.instance().hasOptifine() && ForgeModContainer.forgeLightPipelineEnabled )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method appeng.client.ClientHelper.preinit() orders expressions in a conditional in a sub optimal way [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=fb-contrib%3ASEO_SUBOPTIMAL_EXPRESSION_ORDER)